viewing_party/party.py

```diff
@@ -1,23 +1,156 @@
 # ------------- WAVE 1 --------------------
-
+import copy
+'''
+This function takes in 3 values: the title, genre, and rating of a movie. It uses these three parameters
+and creates a dictionary that includes the title, genre, and rating for the movie. Returns none if any of
+the parameters are "falsy".
+'''
 def create_movie(title, genre, rating):
-    pass
-
-# -----------------------------------------
-# ------------- WAVE 2 --------------------
-# -----------------------------------------
-
-
-# -----------------------------------------
-# ------------- WAVE 3 --------------------
-# -----------------------------------------
-
-        
-# -----------------------------------------
-# ------------- WAVE 4 --------------------
-# -----------------------------------------
-
-# -----------------------------------------
-# ------------- WAVE 5 --------------------
-# -----------------------------------------
-
+    if not title or not genre or not rating:
+        return None
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    return movie    
+'''
+This function takes in 2 values, a dictionary called user_data, and a dictionary called movie. It adds the movie
+as a value to the user_data under the key "watched", which is a list. 
+'''
+def add_to_watched(user_data, movie):
+    user_data_copy = copy.deepcopy(user_data)

```

Comment:
[comment] I'm glad you thought about deep copying the input `user_data`! It's easy to accidentally modify things you shouldn't if you don't. In this case, I don't think deep copying is required, since I think it's expected that you modify the input dictionary. But it's fine to practice not modifying inputs!